test/config.yaml

```diff
@@ -1,4 +0,0 @@
-sampletable: pasilla_sampletable.tsv
-sample_dir: pasilla
-assembly: dm6
-data_dir: /data/LCDB/references
```

Comment:
That can be deleted too. get-data.sh should be generalized anyway once we get more test datasets.

The bigger question is, what should stay in the test dir? Requirements should probably live somewhere else . . . in which case there's prob no reason for the test dir now